resources/systemd/openhab2.service

```diff
@@ -6,13 +6,17 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-EnvironmentFile=/etc/default/openhab2
-EnvironmentFile=/etc/default/openhab2_dirs
+Environment=OPENHAB_HOME=/usr/share/openhab2
+Environment=OPENHAB_CONF=/etc/openhab2
+Environment=OPENHAB_RUNTIME=/usr/share/openhab2/runtime
+Environment=OPENHAB_USERDATA=/var/lib/openhab2
+Environment=OPENHAB_LOGDIR=/var/log/openhab2
+Environment=OPENHAB_USER=openhab
+Environment=OPENHAB_GROUP=openhab
+EnvironmentFile=-/etc/default/openhab2
 User=openhab
 Group=openhab
```

Comment:
2nd option, I construct a template service file so that anyone wanting a custom user would use systemctl start user@openhab2.service